src/functions/last.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * Returns the last element of an array. If anything other than an array is passed in, the input is returned.
+ * @param {any[]} a array of any values
+ * @return {*} The last element of 'a'. Returns `a` if `a` is not an array.
+ *
+ * @example
+ * last(2) // returns 2
+ * last('foo') // returns 'foo'
```

Comment:
Same deal here, I don't want to document the results of what amounts to a side effect.